data/json/items/comestibles/med.json

```diff
@@ -1171,6 +1173,8 @@
     "stim": -10,
     "healthy": -5,
     "fun": 50,
+    "vitamins": [ [ "vit_mme", 1000 ] ],
```

Comment:
The math doesn't add up.
1g of heroin is 1000mg of heroin. If we assume it's only half as effective as morphine (as the comment implies), that's 500 MME, which is 50000 vit_mme, right?